llvm/test/MC/AMDGPU/gfx11_asm_vop3_dpp16.s

```diff
@@ -345,7 +348,7 @@ v_alignbit_b32_e64_dpp v5, v1, v2, vcc_hi row_shr:1
 v_alignbit_b32_e64_dpp v5, v1, v2, vcc_lo row_shr:15
 // GFX11: v_alignbit_b32_e64_dpp v5, v1, v2, vcc_lo row_shr:15 row_mask:0xf bank_mask:0xf ; encoding: [0x05,0x00,0x16,0xd6,0xfa,0x04,0xaa,0x01,0x01,0x1f,0x01,0xff]
 
-v_alignbit_b32_e64_dpp v5, v1, v2, ttmp15 row_ror:1
+v_alignbit_b32_e64_dpp v5, v1, v2, ttmp15 row_ror:1 row_mask:0xf bank_mask:0xf
```

Comment:
You are still adding `row_mask:0xf bank_mask:0xf`.

Reply:
removed